src/app/pages/database-details/database-details.scss

```
@@ -0,0 +1,64 @@
+  min-height: 460px;
+  height: 460px;
+  .chart_div {
+    max-height: 350px !important;
+    height: 350px !important;
+    max-width: 100% !important;
+  }
+}
+
+:host ::ng-deep .block_full {
+  @apply px-6 py-6 bg-secondary border-emerald-400 border border-solid rounded-lg flex flex-col gap-4 relative overflow-hidden;
+  width: calc(100%);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)